### DISCUSSION TITLE
Welcome to Privatezilla Discussions!

### DISCUSSION THREAD

**Discussion by Belim** [mirinsoft/spydish#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vanwaard** [comment]
Is it possible to translate Privatezella in Dutch

**Comment by NotSpammin** [comment]
Hey, Thank you for providing this product! I'm new to using it on my windows 10 home laptop and it seems to work great!
Question? - Will this work with Windows 7 Home?
I have a desktop computer that I almost never let hit the net. Every time I've hit the net MS has crapped on me. they update the os and move everything into folders so I can't find my stuff and screw up my programs so I can't use them (I lost Adobe CS6 due to MS screwing me over!). 
Will this work with Windows 7 Home? - That's my question.

**Comment by blecaplol** [comment]
Sorry does Privatezilla 0.60.0 works with directly with Win 11? (No upgrade)
